### ISSUE TITLE
Remove oci-systemd-hook from README

### ISSUE THREAD

**Issue by tiran** [freeipa/freeipa-container#324]
The Fedora images of freeipa/freeipa-server on Dockerhub seem to be build with ``oci-systemd-hook`` present. The presence of the hooks are breaking FreeIPA container for me. Dan has [retired](https://github.com/containers/podman/issues/4293#issuecomment-545592126) the hooks ``oci-register-machine`` and ``oci-systemd-hook`` but they are not getting removed automatically.

```
# podman --log-level=debug run --replace=true --name freeipa-server-container -ti -h ipa.example.test --read-only  -v /var/lib/ipa-data:/data:Z freeipa/freeipa-server
...
Error: error executing hook `/usr/libexec/oci/hooks.d/oci-systemd-hook` (exit code: 1): OCI runtime error
# podman images
REPOSITORY                        TAG     IMAGE ID      CREATED      SIZE
docker.io/freeipa/freeipa-server  latest  0a1027fc11d4  13 days ago  876 MB
```

```
# journalctl
...
Started libcrun container.
systemdhook <debug>: 54353e27f609: rootfs=/var/lib/containers/storage/overlay/38eff65ebb1820f18d9b5f5984f5db4f3cc3396fa38222d6011404b5135a5a92/merged
systemdhook <debug>: 54353e27f609: gidMappings not found in config
systemdhook <debug>: 54353e27f609: GID: 0
systemdhook <debug>: 54353e27f609: uidMappings not found in config
systemdhook <debug>: 54353e27f609: UID: 0
systemdhook <debug>: 54353e27f609: /run already present as a mount point in container configuration, skipping
systemdhook <debug>: 0::/machine.slice/libpod-54353e27f6091dcf56c091c8ee03d500eb2c0ec7188af215c1b865a1e95aa315.scope/container
: 54353e27f609
systemdhook <debug>: 54353e27f609: ::/machine.slice/libpod-54353e27f6091dcf56c091c8ee03d500eb2c0ec7188af215c1b865a1e95aa315.scope/container
systemdhook <error>: 54353e27f609: Failed to get memory subsystem path for the process: File exists
conmon 54353e27f6091dcf56c0 <ninfo>: console = {.name = '(null)'; .fd = 0}
conmon 54353e27f6091dcf56c0 <nwarn>: Failed to get console terminal settings
libpod-54353e27f6091dcf56c091c8ee03d500eb2c0ec7188af215c1b865a1e95aa315.scope: Succeeded.
conmon 54353e27f6091dcf56c0 <error>: Failed to create container: exit status 255
...
```

**Comment by tiran** [comment]
Ah sorry, I got confused. The hooks are not part of the image. I had ``oci-systemd-hook`` installed in my OS. I have updated my laptop all the way from F29 to F32.

**Comment by tiran** [comment]
Reopening
It would be a good idea to remove ``oci-systemd-hook`` from the README.

**Comment by adelton** [comment]
What is the problem with the `oci-systemd-hook` information in the README? It is still valid infomation about `docker` use cases on RHEL 7 / CentOS 7 vs on Ubuntu.
The fact that we do not have good description of `podman` setups is a different story but we need `podman`-based setups covered in CI before making claims about them. We have https://github.com/freeipa/freeipa-container/issues/186 open about better documenting `podman` workflow.

**Comment by adelton** [comment]
We seem to have lost traction here.

**Comment by adelton** [comment]
To close the loop here -- I've filed https://github.com/freeipa/freeipa-container/pull/342 now to add podman documentation to README. As part of the change I've dropped information about `oci-systemd-hook` because discussion in https://github.com/freeipa/freeipa-container/pull/335 suggests that it is deprecated and can be harmful with podman.
This leaves us with setups on RHEL 7 and CentOS 7 hosts with docker or old podman not documented ... but unless we have a way to run tests on those machines in some CI, I'd probably consider them too old to keep documentation specific for them around.
